### PR TITLE
Add implementation for mobile phones 

### PR DESCRIPTION
**Pull request by ZimnitskiYury**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+            <h2 class="our-advantages__care-title">Забота о клиентах</h2>
+            <span class="our-advantages__care-annotation">Для нас важно</span>
+            <p class="our-advantages__care-content">Нам важно чтобы каждый торт был безупречным, как внутри, так и снаружи, чтобы он был создан исключительно для вас и для вашего события.</p>
+        </div>
+    </section>
+
+    <!-- HOW-TO-ORDER IS FULLTESTED AND WORKED FOR DESKTOP/TABLET/MOBILE -->
+    <section class="order" id="how-to-order">
+        <ol class="order__instruction">
+            <h2 class="order__instruction-header">Как заказать</h2>
```
Comment (drodov) [inline index.html:304]: Keep only list items inside list container.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+            <li class="order__step">
+                <article class="order__step-description">
+                    <span class="order__title">Срок заказа</span>
+                    <p class="order__annotation">Размещать заказы желательно за 2-3 дня. На выходные заказы лучше размещать минимум за неделю.</p>
+                </article>
+            </li>
+            <li class="order__step">
+                <article class="order__step-description">
```
Comment (drodov) [inline index.html:312]: It's not an appropriate tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,423 @@
...
+                    <p class="order__annotation">Доставка за пределами города рассчитывается индивидуально.</p>
+                </article>
+            </li>
+        </ol>
+    </section>
+
+    <!-- CONTACTS IS FULLTESTED AND WORKED FOR DESKTOP/TABLET/MOBILE -->
+    <section class="contacts" id="contacts">
+        <h2 class="contacts__title">Контакты</h2>
+        <span class="contacts__annotation">Остались вопросы?</span>
```
Comment (drodov) [inline index.html:329]: Use `header` tag, please check other places.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,423 @@
...
+
+                <textarea class="feedback-form__input-question" placeholder="Ваш вопрос" name="question" id='question' required></textarea>
+                <div class="feedback-form__button">
+                    <button>Отправить</button>
+                </div>
+            </form>
+            <ul class="contact-info__list">
+                <li class="contact-info__list-item">
+                    <p class="contact-info__phone-headline"> Или позвоните по телефону:</p>
+                    <a class="contact-info__phone-link" href="tel:+79068999296"><i class="fas fa-phone-alt"></i> +7 906 899 92 96 </a>
```
Comment (drodov) [inline index.html:345]: Do you need to use `i` tag?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,423 @@
...
+
+                <input type="text" class="feedback-form__input" placeholder="Email адрес" name="email" id='email' required>
+
+                <textarea class="feedback-form__input-question" placeholder="Ваш вопрос" name="question" id='question' required></textarea>
+                <div class="feedback-form__button">
+                    <button>Отправить</button>
+                </div>
+            </form>
+            <ul class="contact-info__list">
+                <li class="contact-info__list-item">
```
Comment (drodov) [inline index.html:343]: All items look like separate blocks to me.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,423 @@
...
+                </li>
+            </ul>
+        </div>
+    </section>
+
+    <!-- FOOTER IS FULLTESTED AND WORKED FOR DESKTOP/TABLET/MOBILE -->
+    <footer>
+        <div class="footer__menu">
+            <div class="footer__logo">
+                <h5 class="logo__text">Cakes</h5>
```
Comment (drodov) [inline index.html:368]: Let's discuss using a heading here.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,423 @@
...
+            </div>
+
+            <ul class="nav-list">
+                <li class="nav-list__item"><a class="nav-list__link" href="#categories">Наши торты</a></li>
```
Comment (drodov) [inline index.html:372]: Do you need both classes here?

File: index.html [hunk 8]
```diff
@@ -0,0 +1,423 @@
...
+            <div class="footer__social">
+                <a href="https://instagram.com"><img src="./assets/img/social_instagram.png" alt="Наш инстаграм" /></a>
+                <a href="https://vk.com"><img src="./assets/img/social_vk.png" alt="Наш Вк" /></a>
+            </div>
+        </div>
+        <div class="footer__credentials">
+            <hr class="footer__credentials-line">
+            <span class="footer__credentials-copyright">&#169; 2018 Cakes</span>
+        </div>
+        <div class="footer__credentials-tablet">
```
Comment (drodov) [inline index.html:389]: Please check how to name modifiers and use them,

File: index.html [hunk 9]
```diff
@@ -0,0 +1,423 @@
...
+            <hr class="footer__credentials-line">
+            <div class="footer__credentials-socials">
+                <span class="footer__credentials-copyright">&#169; 2018 Cakes</span>
+                <a href="https://instagram.com"><img src="./assets/img/social_instagram.png" alt="Наш инстаграм" /></a>
+                <a href="https://vk.com"><img src="./assets/img/social_vk.png" alt="Наш Вк" /></a>
+            </div>
+        </div>
+        <div class="footer__credentials-mobile">
```
Comment (drodov) [inline index.html:397]: Let's move device-specific classes to separate elements instead of the whole footer.

**Review by drodov — CHANGES_REQUESTED** [review]

File: index.html (hunk 6 above)
Comment (ZimnitskiYury) [reply index.html]: Replaced to <span>

File: index.html (hunk 9 above)
Comment (ZimnitskiYury) [reply index.html]: Deleted

File: index.html (hunk 5 above)
Comment (ZimnitskiYury) [reply index.html]: Fixed

File: css/contacts/contact-info.css [hunk 10]
```diff
@@ -0,0 +1,134 @@
+.contact-info__list {
+    display: flex;
+    flex-direction: column;
+
+    width: 344px;
+    padding-top: 26px;
+
+    align-items: flex-start;
+    justify-content: flex-start;
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply css/contacts/contact-info.css]: Deleted

File: css/contacts/contact-info.css [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+    width: 60px;
+    height: 60px;
+    margin-right: 30px;
+
+    border-radius: 30px;
+    background-color: white;
+    background-image: url("../../assets/svg/instagram-logo.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 30px;
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply css/contacts/contact-info.css]: Combined

File: css/about-us/about-us.css [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+    }
+}
+
+@media (max-width:767px) {
+    .about-us {
+        flex-direction: column;
+
+        width: 87.5%;
+        height: auto;
+        padding: 48px 6.25% 0 6.25%;
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply css/about-us/about-us.css]: Fixed

File: css/about-us/about-us.css [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+    .about-us__title {
+        font-size: 28px;
+        line-height: auto;
+    }
+    .about-us__sub-header {
+        font-size: 26px;
+    }
+    .about-us__dictum-title {
+        font-size: 15px;
+    }
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply css/about-us/about-us.css]: Fixed everywhere. `line-height: auto;` deleted (wrong property)

File: css/categories.css [hunk 14]
```diff
@@ -0,0 +1,281 @@
...
+        width: 606px;
+        height: 580px;
+        margin-left: -212px;
+
+        background-color: #f3f3f5;
+        box-shadow: 0 10px 12.74px 0.26px rgba(163, 6, 60, 0.05);
+
+        align-items: center;
+    }
+}
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply css/categories.css]: fixed everywhere

File: css/categories.css [hunk 15]
```diff
@@ -0,0 +1,281 @@
...
+
+.categories__name {
+    z-index: 1;
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply css/categories.css]: Image overlaps text

File: css/categories.css [hunk 16]
```diff
@@ -0,0 +1,281 @@
...
+
+        letter-spacing: 0.28px;
+
+        color: #2d2829;
+
+        font-family: "Playfair Display Bold";
+        font-size: 28px;
+        line-height: 40px;
+
+        align-self: flex-start;
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply css/categories.css]: Cleaned everywhere

File: css/categories.css [hunk 17]
```diff
@@ -0,0 +1,281 @@
...
+
+        align-items: center;
+    }
+    .categories__container-item:last-of-type {
+        display: flex;
+        flex-direction: column;
+
+        width: 606px;
+        height: 580px;
+        margin-left: -212px;
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply css/categories.css]: Fixed

File: css/categories.css [hunk 18]
```diff
@@ -0,0 +1,281 @@
...
+        padding: 14px 4.28%;
+    }
+    .categories__container-item:first-of-type {
+        width: 91.44%;
+        height: 180px;
+        margin-right: 0;
+        margin-bottom: 10px;
+        padding: 14px 4.28%;
+    }
+    .categories__container-item:last-of-type {
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply css/categories.css]: Fixed

File: css/contacts/contact-info.css [hunk 19]
```diff
@@ -0,0 +1,134 @@
...
+
+    font-family: Circe;
+    font-size: 20px;
+}
+
+.contact-info__email-link {
+    color: #2d2829;
+
+    font-family: Circe;
+    font-size: 20px;
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply css/contacts/contact-info.css]: Fixed everywhere

File: css/contacts/contact-info.css [hunk 20]
```diff
@@ -0,0 +1,134 @@
...
+    }
+}
+
+@media (max-width:767px) {
+    .contact-info__list {
+        width: 100%;
+        padding-top: 28px;
+    }
+    .contact-info__list-item {
+        margin-bottom: 22px;
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply css/contacts/contact-info.css:34]: Duplicates removed

File: css/footer/footer.css [hunk 21]
```diff
@@ -0,0 +1,262 @@
+@import url("./nav-list.css");
+footer {
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply css/footer/footer.css]: Changed to class

File: index.html [hunk 22]
```diff
@@ -0,0 +1,423 @@
...
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/06f4c9154d.js" crossorigin="anonymous"></script>
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.10.1/jquery.min.js"></script>
+</head>
+
+<body>
+    <!-- HEADER IS FULLTESTED AND WORKED FOR DESKTOP/TABLET/MOBILE -->
+    <header class="header">
+        <div class="header__logo">
+            <a href="./index.html" class="header__logo-link">Cakes</a>
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply index.html]: fixed

File: index.html [hunk 23]
```diff
@@ -0,0 +1,423 @@
...
+            <li><a class="nav-menu__link" href="#how-to-order">Как заказать</a></li>
+            <li><a class="nav-menu__link" href="#contacts">Контакты</a></li>
+        </ul>
+        <a href="tel:+7 906 899 92 96" class="header__phone"><i class="fas fa-phone-alt"></i> +7 906 899 92 96</a>
+    </header>
+
+    <!-- MAIN IS FULLTESTED AND WORKED FOR DESKTOP/TABLET/MOBILE -->
+    <main class="main-page">
+        <h1 class="main-page__title">Торты <br/>на заказ</h1>
+        <span class="main-page__annotation">Ручная работа</span>
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply index.html]: Added header tag

File: index.html [hunk 24]
```diff
@@ -0,0 +1,423 @@
...
+            </li>
+        </ul>
+    </main>
+
+    <!-- CATEGORIES IS FULLTESTED AND WORKED FOR DESKTOP/TABLET/MOBILE -->
+    <section class="categories" id="categories">
+        <h2 class="categories__title">Наши торты</h2>
+
+        <ul class="categories__container">
+            <li class="categories__container-item">
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply index.html:62]: Did

File: index.html [hunk 25]
```diff
@@ -0,0 +1,423 @@
...
+                </div>
+            </a>
+            <article class="dictum">
+                <div class="dictum__user-photo">
+                    <img src="./assets/img/reviews-persone.png" alt="reviews-persone" height="100%" width="100%">
+                </div>
+                <div class="dictum__content">
+                    <header class="dictum__header">
+                        <h1 class="dictum__user">Мария</h1>
+                        <h2 class="dictum__user-info">юрист, 33 года</h2>
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply index.html]: Fixed

File: index.html [hunk 26]
```diff
@@ -0,0 +1,423 @@
...
+            </div>
+        </div>
+        <div class="about-us__profile-photo">
+            <img src="./assets/img/about-us.png" alt="about-us" width="100%" height="100%">
+        </div>
+    </section>
+
+    <!-- OUR-COMMENT IS FULLTESTED AND WORKED FOR DESKTOP/TABLET/MOBILE -->
+    <section class="our-comment">
+        <span class="our-comment__title">’’</span>
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply index.html]: Did it with :before

File: index.html [hunk 27]
```diff
@@ -0,0 +1,423 @@
...
+    </section>
+
+    <!-- OUR-ADVANTAGES IS FULLTESTED AND WORKED FOR DESKTOP/TABLET/MOBILE -->
+    <section class="our-advantages">
+        <h2 class="our-advantages__title">Наши преимущества</h2>
+        <span class="our-advantages__annotation">Почему нас выбирают ?</span>
+        <ul class="our-advantages__list">
+            <li class="our-advantages__list-item">
+                <img class="our-advantages__list-item-image" src="./assets/img/natural-prod.png" alt="natural-product">
+                <article class="our-advantages__advantage">
```
[the remark this answers is not in the document]
Comment (ZimnitskiYury) [reply index.html]: Fixed

File: index.html (hunk 3 above)
Comment (ZimnitskiYury) [reply index.html]: Did

**Pull request merged**